### DISCUSSION TITLE
How many themes can be supported by `ThemeResource`?

### DISCUSSION THREAD

**Discussion by jeanplevesque** [unoplatform/uno#15066] [Q&A] [answered]
Let's say I want to create an app that has more than just Light, Dark, and HighContrast themes (like different colors/moods). Is it possible to define and use more ThemeDictionaries?

**Comment by jeromelaban** [comment, marked as answer]
`ThemeResources` only support `Light`, `Dark` and `HighContrast` in the same way WinUI does. 

If custom themes need to be supported, theme resources can be overridden at the `app.xaml` level explicitly, for all the used resources in all controls templates used by app. Changing the resources requires re-creating pages/user controls for the new theme to be applied.
